### DISCUSSION TITLE
Nano Adblocker i Defender dla Chrome został sprzedany

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#17251] [General]
:penguin: sprzedał Turkom Nano Adblockera i Defendera, bo brakowało mu już czasu na rozwój obu rozszerzeń, nie nadążał za gorhillem. Co o tym sądzicie, będzie lepiej czy gorzej?

Źródło: https://github.com/NanoAdblocker/NanoCore/issues/362

**Comment by MajkiIT** [comment]
no może być ciekawie o ile oni tego nie spieprzą. Bo spuściznę im zostawił.

**Comment by hawkeye116477** [comment]
Ważniejsze rzeczy to najwyraźniej jakaś tajemnica państwowa, a to nie nastraja optymizmem. Zaraz pewnie się okaże, że ci tajemniczy deweloperzy, to gargulec i spółka z proxy wraz z tureckim nickiem albo ktoś powiązany z Betafishem i jedynie dodadzą listę odblokowującą akceptowalne reklamy i ładną stronkę z prośbą o wsparcie...

---------------------------------------------------------------------------------------------
@gargul03 Bu doğru mu çirkin yaratık?

**Reply by MajkiIT** [reply]
No poczekajmy na rozwój sytuacji, ale jak tak przedstawiasz sytuację to tak może być. Skoro komuś to oddał to chyba w dobre ręce.

**Reply by hawkeye116477** [reply]
> Skoro komuś to oddał to chyba w dobre ręce.

No oby, choć gorhill najwyraźniej uważa podobnie jak ja. Już nawet usunęli link do polityki prywatności.
Niby nazywają się `aymaksemagul` i  `altuncunizametdin`, na necie 0 informacji o nich, :penguin: mówi, że oni są freelancerami i są nowi w tymi fachu.

**Reply by krystian3w** [reply]
Link niby wyparował, bo oni nie mają podpiętego do konta.

**Comment by xxcriticxx** [comment]
nie wygląda to za dobrze trzeba ten extension odinstalowac

**Comment by hawkeye116477** [comment]
Stronkę zrobili => https://dev-nano.com, tzn będzie gotowa dopiero za jakieś 35 dni...

**Reply by krystian3w** [reply]
O dziwne, godzinę temu nie było licznika lub http mają źle skonfigurowane (bez przekierowania).

**Comment by xxcriticxx** [comment]
co teraz zamiast nano blockera montujemy?

**Reply by krystian3w** [reply]
Filtry: https://github.com/bogachenko/fuckfuckadblock/ - wada, nadużywanie `generichide` (białe luki po reklamach, wyciekający social/cookies/RODO).

Dodatek/ki: Niby można zostać tak długo aż nie pojawi się wersja z pomysłami tureckimi.

**Reply by hawkeye116477** [reply]
Tyle, że @xxcriticxx pytał się o Nano Adblocker.... W takim przypadku zamiast niego uBlock Origin :smile:.

Jeżeli chodzi o Defendera, to gorhill poleca zgłaszać do uAssets, czyli teoretycznie domyślne listy powinny wystarczyć...

**Reply by krystian3w** [reply]
skraczacze linków "do zarabiania" z tym samym anty-adblock niby wyrastają jak grzyby po deszczu.

**Reply by xxcriticxx** [reply]
sorry napisalem blocker ale chodzilo mi o defendera

**Reply by hawkeye116477** [reply]
No to gorhill każe zgłaszać wszystko na https://github.com/uBlockOrigin/uAssets/issues, twierdzi, że Defender nie potrzebny, sam uBO z własnymi listami da radę, no zobaczymy, bo w przeszłości chyba różnie z tym bywało, choć teraz niby poszerzył też arsenał... Jednak mimo to zazwyczaj Defender chyba miewał jakby trwalsze metody.

**Reply by xxcriticxx** [reply]
dla mnie sie zdaje ze na easylist forum szybciej pomagaja. w uassets jest template do wypelienia

**Reply by hawkeye116477** [reply]
A, leniuszek... Nie ma tak dużo do wypełniania, część możesz zawsze zapisać w jakimś notatniku i potem tylko podmiankę zrobić.

Jeżeli chodzi o EasyList, to ja mam odmienne doświadczenia, choć nie na forum...

**Reply by krystian3w** [reply]
Mogli by ten formularz jakiś interaktywny zrobić (MS na GitHub) a nie jedno wielkie textarea.

**Comment by hawkeye116477** [comment]
Już Turki zaczęli działać => https://github.com/nenodevs/uBlockProtector/commit/944434bd5e2015128f1f1a4adff8d05150c034d8, skopiowali skrypt z https://cdnjs.cloudflare.com/ajax/libs/socket.io/2.3.1/socket.io.js, ale usunęli informacje o prawach autorskich, czyli złamali postanowienia licencyjne.

Oprócz tego są jakieś jeszcze inne zmiany, których nie ma na GH.

https://github.com/NanoAdblocker/NanoCore/issues/362#issuecomment-709428210

Ciekawostka: Nazwali się N**e**nodevs zapewne dlatego, że nazwa N**a**nodevs była już zajęta przez panów od kryptowalut...

Ostateczny wniosek jest taki, że **pora odinstalować Nano Defender dla Chromowatych**.

**Reply by xxcriticxx** [reply]
@hawkeye116477 ja juz to zrobilem dwa tygodnie temu

**Comment by FadeMind** [comment]
Widzę, że panowie na bieżąco ;) Zostawiam:

Analiza gorhilla: `https://github.com/NanoAdblocker/NanoCore/issues/362#issuecomment-709428210`

kontynuacja wątku: `https://github.com/jspenguin2017/Snippets/issues/2`

Reddit: https://www.reddit.com/r/chrome/comments/jc0vms/nanoadblocker_nanodefender_is_malware_now/

tl;dr with a bit of context: The uBlock Origin developer, gorhill, looked into it. It seems to send information on every network connect, purpose is unknown. Nobody even knows really who those developers are. He suggests removing the extension as it can be considered malware now

**Reply by hawkeye116477** [reply]
Już zdjęli Nano Defendera ze sklepu, ~ale Nano Adblocker jeszcze jest~.

**Comment by hawkeye116477** [comment]
Wersja Nano Defendera dla Firefoksa prawdopodobnie będzie nadal kontynuowana przez panią LiCyborę, tylko że już pod nową nazwą.

https://www.ghacks.net/2020/10/16/time-to-remove-nano-adblocker-and-defender-from-your-browsers-except-firefox/

https://github.com/LiCybora/NanoDefenderFirefox/issues/187
